distributions/distribution-resources/src/main/resources/deb/systemd/openhab2.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=Starts and stops the openHAB Home Automation Bus
-Documentation=http://www.openhab.org
+Description=openHAB 2 Home Automation Bus Solution
```

Comment:
This actually was inspired by you, telling me at some point that you like to call openHAB a "solution" :) I'm okay with your suggestion, it's the one used on the webpages. I feel however like this should follow the rule of uppercase in headlines
"openHAB 2 - Empowering the Smart Home" or
"openHAB 2 - Empowering the SmartHome"
 Yay or Nay?